docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   # TODO: change this name to match the name of your app
-  expressjs-template:
+  ra:
```

Comment:
is this ra or cui-ra?

Reply:
The name was changed by the bin/init.sh script. Should i change this to cui-ra ?